app/models/Document.js

```diff
@@ -44,7 +44,11 @@ export default class Document extends BaseModel {
 
   @action
   updateTitle() {
-    set(this, parseTitle(this.text));
+    const { title, emoji } = parseTitle(this.text);
+
+    if (title) {
```

Comment:
Prevent an empty `title` value overriding the `Untitled` value sent from the server